docs/ops/movement/ScatterElementsUpdate_12.md

```diff
@@ -0,0 +1,209 @@
+
+For instance, in a 3D tensor case, the update of the element corresponding to the ``[i][j][k]`` is performed as below:
+
+- reduction == "copy"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] = updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] = updates[i][j][k] if axis = 2
+
+
+- reduction == "sum"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] += updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] += updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] += updates[i][j][k] if axis = 2
+
+
+- reduction == "prod"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] *= updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] *= updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] *= updates[i][j][k] if axis = 2
+
+
+- reduction == "min"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = min(updates[i][j][k], output[indices[i][j][k]][j][k]) if axis = 0,
+    output[i][indices[i][j][k]][k] = min(updates[i][j][k], output[i][indices[i][j][k]][k]) if axis = 1,
+    output[i][j][indices[i][j][k]] = min(updates[i][j][k], output[i][j][indices[i][j][k]]) if axis = 2
+
+
+- reduction == "max"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = max(updates[i][j][k], output[indices[i][j][k]][j][k]) if axis = 0,
+    output[i][indices[i][j][k]][k] = max(updates[i][j][k], output[i][indices[i][j][k]][k]) if axis = 1,
+    output[i][j][indices[i][j][k]] = max(updates[i][j][k], output[i][j][indices[i][j][k]]) if axis = 2
+
+
+**Attributes**:
+
+* *reduction*
+
+  * **Description**: The type of operation to perform on the inputs.
+  * **Range of values**: one of ``copy``, ``sum``, ``prod``, ``min``, ``max``
+  * **Type**: `string`
+  * **Default value**: ``copy``
+  * **Required**: *no*
+
+
+* *use_init_val*
+
+  * **Description**: Controls whether the elements in the data input tensor are used as init value for reduce operations.
+  * **Range of values**:
+    * true - data input elements are used
+    * false - data input elements are not used
+  * **Type**: boolean
+  * **Default value**: true
+  * **Required**: *no*
+  * **Note**: The attribute has no effect for *reduction* == "copy"
+
+
+**Inputs**:
+
+*   **1**: ``data`` tensor of arbitrary rank ``r`` and of type *T*. **Required.**
+
+*   **2**: ``indices`` tensor with indices of type *T_IND*. The rank of the tensor is equal to the rank of ``data`` tensor. All index values are expected to be within bounds ``[-d, d - 1]`` along dimension ``d`` pointed by ``axis``. If multiple indices point to the
+same output location then the order of updating the values is undefined. If an index points to non-existing output tensor element or is negative then exception is raised. **Required.**
```

Comment:
why is the exception is raised if negative index? Should not we normalize it?

Reply:
Also, need to describe how to normalize negative indices. May be I missed it here:)

Reply:
Good catch, it was leftover from the previous version of the op. Updated.